test/r401_test.js

```diff
@@ -0,0 +1,336 @@
+const patientShawnee = require('./fixtures/r4/Johnnie679_Hermiston71_2cd30bd6-3a87-4191-af90-6daa70f58f55.json');
+
+describe('#FHIRWrapper_R4 v4.0.1', () => {
+  let fhirWrapper;
+  let conditionResourceWithNoType;
+  let domainResource;
+  before(() => {
+    fhirWrapper = cqlfhir.FHIRWrapper.FHIRv401();
+    conditionResourceWithNoType = JSON.parse(JSON.stringify(conditionResource));
+    delete conditionResourceWithNoType.resourceType;
+    domainResource = JSON.parse(JSON.stringify(conditionResource));
+    domainResource.resourceType = 'DomainResource';
+  });
+
+  it('should wrap a fhir resource to the correct type when type not specified', () => {
+    let fhirObject = fhirWrapper.wrap(conditionResource);
+    expect(fhirObject.getTypeInfo().name).to.equal('Condition');
+    // Check one Condition property to be sure
+    const recordedDate = fhirObject.getDate('recordedDate');
+    expect(recordedDate.isDateTime).to.be.true;
+    expect(recordedDate).to.deep.equal(cql.DateTime.parse('2013-04-04'));
+  });
+
+  it('should wrap a fhir resource to the type specified if upcasting', () => {
+    // inheritance is: Condition < DomainResource < Resource
+    let fhirObject = fhirWrapper.wrap(conditionResource, 'DomainResource');
+    expect(fhirObject.getTypeInfo().name).to.equal('DomainResource');
+    // Check one DomainResource property to be sure
+    const textStatus = fhirObject.get('text.status');
+    expect(textStatus.getTypeInfo().name).to.be.equal('NarrativeStatus');
+    expect(textStatus.value).to.equal('generated');
+
+    fhirObject = fhirWrapper.wrap(conditionResource, 'Resource');
+    expect(fhirObject.getTypeInfo().name).to.equal('Resource');
+    // Check one Resource property to be sure
+    const id = fhirObject.get('id');
+    expect(id.getTypeInfo().name).to.be.equal('id');
+    expect(id.value).to.equal('f201');
+  });
+
+  it('should wrap a fhir resource to the type specified if downcasting', () => {
+    // inheritance is: Condition < DomainResource < Resource
+    let fhirObject = fhirWrapper.wrap(domainResource, 'Condition');
+    expect(fhirObject.getTypeInfo().name).to.equal('Condition');
+    // Check one Condition property to be sure
+    const recordedDate = fhirObject.getDate('recordedDate');
+    expect(recordedDate.isDateTime).to.be.true;
+    expect(recordedDate).to.deep.equal(cql.DateTime.parse('2013-04-04'));
+  });
+
+  it('should error if requested type is incompatible', () => {
+    expect(function(){fhirWrapper.wrap(conditionResource, 'Observation');}).to.throw('Incompatible types: FHIR resourceType is Condition which cannot be cast as Observation');
+  });
+
+  it('should wrap a fhir resource to the type specified if real type unknown', () => {
+    let fhirObject = fhirWrapper.wrap(conditionResourceWithNoType, 'Observation');
+    expect(fhirObject.getTypeInfo().name).to.equal('Observation');
+  });
+});
+
+describe('#R4', () => {
```

Comment:
Can you update this to:
```js
describe('#R4 v4.0.1', () => {
```
like you did for the FHIRWrapper test above?  That way it's easier to distinguish failing 4.0.1 tests from failing 4.0.0 tests.